include/vcpkg/base/messages.h

```diff
@@ -1003,6 +1011,14 @@ namespace vcpkg
                     "msiexec failed while extracting '{path}' with launch or exit code {exit_code} and message:");
     DECLARE_MESSAGE(NoLocalizationForMessages, (), "", "No localized messages for the following: ");
     DECLARE_MESSAGE(NoRegistryForPort, (msg::package_name), "", "no registry configured for port {package_name}");
+    DECLARE_MESSAGE(PackageFailedtWhileExtracting,
+                    (msg::package_name, msg::path),
+                    "",
+                    "'{package_name}' failed while extracting '{path}'.");
+    DECLARE_MESSAGE(PackageFailedtWhileExtractingWithMessage,
+                    (msg::package_name, msg::path, msg::error),
+                    "'{error}' is the error message.",
+                    "'{package_name}' failed while extracting '{path}' with message '{error}'.");
```

Comment:
I would like to see this one deleted and just have the message appended if present.